cmd/kor/all.go

```diff
@@ -11,7 +13,11 @@ var allCmd = &cobra.Command{
 	Args:  cobra.ExactArgs(0),
 	Run: func(cmd *cobra.Command, args []string) {
 		if outputFormat == "json" {
-			kor.GetUnusedAllJSON(namespace, kubeconfig)
+			res, _ := kor.GetUnusedAllJSON(namespace, kubeconfig)
+			fmt.Println(res)
+		} else if outputFormat == "yaml" {
+			res, _ := kor.GetUnusedAllYAML(namespace, kubeconfig)
```

Comment:
I don't think a separate function is needed, we can rename `GetUnusedAllJSON` to something like `GetUnusedAllStructured` (or something else) and pass the desired output format.